experimental/msal-angular-v2/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "msal-angular-v2",
```

Comment:
```suggestion
  "name": "msal-angular-v2",
```

Reply:
FYI this specific package.json file is for the workspace, and won't be published.